errors/calcWithExceptions/solutionB/src/main/java/oointro/Runner.java

```diff
@@ -0,0 +1,83 @@
+package oointro;
+
+public class Runner{
+  public static void main(String[] args){
+    // Test 1: add, subtract, multiply, divide all work normally
+    System.out.println("Testing add, subtract, multiply, divide with normal operations");
+    Calculator calcOne = new Calculator();
+
+    System.out.println("Testing add");
+    calcOne.add(500);
+    calcOne.add(-600);
+
+    System.out.println("Testing subtract");
+    calcOne.subtract(200);
+    calcOne.subtract(-200);
+
+    System.out.println("Testing multiply");
+    calcOne.multiply(-3);
+    calcOne.multiply(1/2);
+
+    System.out.println("Testing divide");
+    calcOne.divide(5);
+    calcOne.divide(-0.5);
+
+    // Test 2: Division by zero
+    System.out.println("Testing division by zero");
+    Calculator calcTwo = new Calculator();
+    calcTwo.add(100);
+    try {
+      calcTwo.divide(0.0);
+    } catch (Exception e) {
+      System.out.println("Division by zero caught");
+    }
+
+    // Test 3: Overflows
+    Calculator calcThree;
+    
```

Comment:
this is very good.   If you want to fine-tune your exception handling even more,  catch each exception separately with its own catch block.  In this simple example that doesn't make much sense but in larger applications it is often very useful.   

You don't need to change this code unless you want to .  It is fine as is.

Reply:
Ok 👍, I refined all caught exceptions `ValueOverflowException` and `DivisionByZeroException`